New requirements page hosted 2 (#1914)
mendersoftware/mender-docs: 02.Overview/16.Requirements/docs.md

---
title: Requirements
taxonomy:
    category: docs
---

## Working with Firewalls

> Corporate Firewalls must permit the following URLs to the devices and for letting Hosted Mender works correctly:

Reviewer: Nit:

```suggestion
Corporate Firewalls must permit the following URLs to the devices in order for Hosted Mender to work correctly:
```

**Hosted Mender access**: Devices, APIs and browser access
* `https://hosted.mender.io`
* `https://*.hosted.mender.io`
* `https://eu.hosted.mender.io`
* `https://*.eu.hosted.mender.io`

**Artifact storage access**: Devices, APIs and browser access
* `https://s3.amazonaws.com/hosted-mender-artifacts` - Amazon S3 Path-style request

> * `https://s3.amazonaws.com/hosted-mender-artifacts-eu` - Amazon S3 Path-style request

Reviewer 2: As I mentioned in the other PR, these will need to be replaced with Azure blob storage URLs - they use a virtual-host URLs.

Author: But why they need to be replaced? They'll continue to work with those URLs for the US instance, don't they?

About the new URLs for Azure Blob storage, as for now I don't know the address, that's why I'm not including it. As soon as the new Azure Blob storage is ready for the new EU instance, I'll append in the documentation.

Reviewer 2: > About the new URLs for Azure Blob storage, as for now I don't know the address, that's why I'm not including it. As soon as the new Azure Blob storage is ready for the new EU instance, I'll append in the documentation.

But there are s3 URL pointing to hosted-mender-artifacts-eu which does not exist. I would remove the two URLs from this PR and add them once the EU instance is in a production ready state.

Author: These hosted-mender-artifacts-eu are the EU storage for the current US instance, currently used only from a single customer:
https://artifacts-eu.hosted.mender.io/minio/login

Reviewer 2: Ohh, right! I thought it was the tentative bucket name for the upcoming EU instance. Sorry about the confusion.
